e2e/stress/IoTClientPerf/readme.md

```diff
@@ -0,0 +1,58 @@
+# Azure IoT Hub Stress Test
+
+Use `dotnet run` to display the help page for this tool.
+
+## Prerequisites
+
+##### Create an IoT Hub for performance testing.
+##### Create one IoT Hub device with SAS token and one with X509 certificate authentication.
+
+Fill in the following environment variables (see the e2e\test prerequisites for details):
+```
+IOTHUB_CONN_STRING_CSHARP=
+IOTHUB_DEVICE_CONN_STRING=
+IOTHUB_DEVICE_CONN_STRING2=
+IOTHUB_X509_PFX_CERTIFICATE=
+```
+
+##### Provision sufficient devices for the hub:
+
+`dotnet run -- -a sas -n 100 -o out.csv -f generate_iothub_config`
+
+Follow the instructions in the output for how to import the `devices.txt` file into your hub.
```

Comment:
Note: You will need the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) and the [IoT extension for Azure CLI](https://github.com/Azure/azure-iot-cli-extension#installation).